daemon/algod/api/algod.oas2.json

```diff
@@ -1770,23 +1814,42 @@
       "type": "object",
       "required": [
         "asset-id",
+        "creator",
```

Comment:
I think that for algod, we don't want to force a creator for an AssetHolding

Reply:
I discovered that even the indexer is not filling in this value (it's an empty string with a TODO) and I removed it from the indexer spec in https://github.com/algorand/indexer/pull/872